src/lib/core/match-media/match-media.ts

```diff
@@ -38,59 +38,78 @@ export class MatchMedia {
     return !!mql ? mql.matches : false;
   }
 
+  /**
+   * External observers can watch for all (or a specific) mql changes.
+   *
+   * If a mediaQuery is not specified, then ALL mediaQuery activations will
+   * be announced.
+   */
+  observe(): Observable<MediaChange>;
+  observe(mqList: string[]): Observable<MediaChange>;
+
   /**
    * External observers can watch for all (or a specific) mql changes.
    * Typically used by the MediaQueryAdaptor; optionally available to components
    * who wish to use the MediaMonitor as mediaMonitor$ observable service.
    *
-   * NOTE: if a mediaQuery is not specified, then ALL mediaQuery activations will
-   *       be announced.
+   * Use lazy registration...
    */
-  observe(mediaQuery?: string): Observable<MediaChange> {
-    if (mediaQuery) {
-      this.registerQuery(mediaQuery);
-    }
-
-    return this._observable$.pipe(
-      filter(change => (mediaQuery ? (change.mediaQuery === mediaQuery) : true))
+  observe(mqList?: string[]): Observable<MediaChange> {
+    const matchMedia$ = this._observable$.pipe(
+        filter((change: MediaChange) => {
+          return !!(mqList || []).find((query: string) => change.mediaQuery === query);
```

Comment:
We should use `Array.includes` here. And if we can't, we should figure out why our TS config won't let us.